app/mailers/application_mailer.rb

```diff
@@ -1,5 +1,5 @@
 class ApplicationMailer < ActionMailer::Base
   domain = ENV.fetch('MAILGUN_DOMAIN', 'roqua.nl')
-  default from: "noreply@#{domain}"
+  default from: "CATja Screening <noreply@#{domain}>"
```

Comment:
Prima om geen spatie in bestandsnaam te doen en dit later aan opdrachtgever te vragen.

Ik stelde voor om het nu aan te passen naar `CATja-screening` (mét streepje, ook in de afzendernaam) omdat dat de Nederlandse spellingsregels beter volgt.